commons-test/src/main/java/org/infinispan/commons/test/PolarionJUnitXMLReporter.java

```diff
@@ -189,7 +190,8 @@ protected void generateReport(ISuite suite) {
       document.pop();
 
       // Reset output directory
-      Utils.writeUtf8File(suite.getOutputDirectory().replaceAll("/Surefire suite", ""),
+      Utils.writeUtf8File(
+            suite.getOutputDirectory().replaceAll(String.format("%sSurefire suite", File.pathSeparator), ""),
```

Comment:
Actually, since replaceAll takes a regex as the first parameter you could write:
`Utils.writeUtf8File(suite.getOutputDirectory().replaceAll(".Surefire` suite", "")
instead